packages/create-fuels/scripts/rewriteTemplatePackageJson.ts

```diff
@@ -0,0 +1,13 @@
+import { versions } from '@fuel-ts/versions';
+import { readFileSync, writeFileSync } from 'fs';
+import { join } from 'path';
+
+const filepath = join(__dirname, '../templates/nextjs/package.json');
+
+let contents = readFileSync(filepath, 'utf-8');
+
+contents = contents.replace(/xprebuild/g, 'prebuild');
+contents = contents.replace(/"fuels": "workspace:\*"/, `"fuels": "${versions.FUELS}"`);
+contents = contents.replace(/"build": "pnpm run xprebuild && next build"/, `"build": "next build"`);
```

Comment:
Does this work? Line 9 replaces `xprebuild` with `prebuild` globally, but this one still has `xprebuild` in it (`...xprebuild && next build`)

Reply:
It works, this is the output in the template:
<img width="411" alt="image" src="https://github.com/FuelLabs/fuels-ts/assets/39617427/9e21e927-96e9-4c19-9979-cddf48b73f5f">

Reply:
🕵️